CMakeLists.txt

```diff
@@ -184,13 +229,18 @@ if (NOT USE_XSDK_DEFAULTS)
     endforeach()
     if (Tasmanian_ENABLE_MATLAB)
         message(" -D Tasmanian_MATLAB_WORK_FOLDER:PATH=${Tasmanian_MATLAB_WORK_FOLDER}")
-        message(" pre-install MATLAB folder: addpath('${CMAKE_BINARY_DIR}/matlab/')")
+        message(" pre-install MATLAB folder: addpath('${CMAKE_CURRENT_BINARY_DIR}/MATLAB/matlab/')")
```

Comment:
Why did you change to `CMAKE_CURRENT_BINARY_DIR`?  Why did you add a "MATLAB" directory?

Reply:
All paths are relative now, so Tasmanian can be added with `add_subdir()`.

The MATLAB folder contain the scripts so those can be used during build and before install. This is done for testing and sometimes convenience. Double MATLAB may be redundant, I'll think about it.